charger/vehicle_controlled_charger.go

```diff
@@ -151,7 +137,13 @@ func (c *VehicleControlledCharger) Enable(enable bool) error {
 		return err
 	}
 	c.enabled = enable
-	c.isCharging = enable
+	// reset vehicle status cache
+	//  - delayed to allow vehicle APIs to reflect new charging status
+	go func() {
```

Comment:
This would realistically happen about every minute or so. Not sure that's a good idea. If it is- what's the 30s wait for?

Reply:
> This would realistically happen about every minute or so
Are you referring to the cache expiring or Enable() getting called?
The cache expiry for the VW ID APIs (the ones that I have been testing with) is 15 minutes. 

The wait is there in order to make sure that the Vehicle Status has actually updated and is reflected in the manufacturer's APIs by the time that the cache gets refreshed.

Reply:
Das ist eine race condition- geht nur mit Mutex. Warum muss darauf überhaupt gewartet werden?

Reply:
Um die Race condition zu umgehen, würde ich vorschlagen mit Timern zu arbeiten:

<img width="695" height="444" alt="Screenshot 2025-09-03 at 19 14 08" src="https://github.com/user-attachments/assets/0512b988-4bd1-4e9c-ac41-b2ff2cf5a013" />

Reply:
Ich habe einen neuen commit gemacht, der diese Änderung umsetzt. Damit sollte die Race Condition umgangen sein.